tests/DeepCopyTest/TypeFilter/Spl/SplDoublyLinkedListFilterTest.php

```diff
@@ -25,6 +25,5 @@ public function test_it_deep_copies_a_doubly_linked_spl_list()
         $newList = $filter->apply($list);
 
         $this->assertCount(1, $newList);
-        $this->assertNotSame($foo, $newList->next());
```

Comment:
This was a dead assertion as [`SplDoublyLinkedList::next`](http://php.net/manual/en/spldoublylinkedlist.next.php) does not return anything.